### DISCUSSION TITLE
Remove whitespace in Markdown

### DISCUSSION THREAD

**Discussion by MBurchard** [antfu/eslint-config#474] [Q&A] [answered]
Hi @antfu,

Thank you very much for this project, it offers a lot of useful defaults and has made my old ESLint configurations almost completely superfluous.  
That's the rest of the customisations I use.

```javascript
import antfu from '@antfu/eslint-config';

export default antfu({
  ignores: [
    '*.svg',
  ],
  stylistic: {
    semi: true,
  },
}, {
  rules: {
    'curly': 'off',
    'regexp/strict': 'off',
    'style/brace-style': ['error', '1tbs'],
    'style/max-len': ['warn', {code: 120}],
    'style/object-curly-spacing': ['error', 'never'],
    'style/operator-linebreak': ['error', 'after'],
  },
}, {
  files: ['**/*.md'],
  rules: {
    'no-trailing-spaces': 'off',
    'style/max-len': 'off',
  },
});

```

ESLint always removes the two spaces at the end of lines that are used to create a line break in MD files.
I've tried `'no-trailing-spaces': 'off',` but that obviously didn't do it.
How can I fix it?
Would this fix be something for everyone and should be standard for MD files?

**Comment by Dimava** [comment]
It's `[style/no-trailing-spaces](https://eslint.style/rules/js/no-trailing-spaces)`
![image](https://github.com/antfu/eslint-config/assets/20068737/cd385116-f120-40ab-bc3e-9573b440d215)

I agree that it should be disabled for Markdown though

**Reply by MBurchard** [reply, marked as answer]
🤦‍♂obviously, thank you very much for the correction @Dimava

**Comment by Dimava** [comment]
Requested https://github.com/eslint-stylistic/eslint-stylistic/issues/397
